mailpoet/assets/js/src/segments/dynamic/list/bulk-actions.tsx

```diff
@@ -0,0 +1,202 @@
+import { useState } from 'react';
+import { __, _n, sprintf } from '@wordpress/i18n';
+import { SelectControl } from '@woocommerce/components';
+import { Button, Modal } from '@wordpress/components';
+import { dispatch, select } from '@wordpress/data';
+import { MailPoet } from '../../../mailpoet';
+import { DynamicSegment } from '../types';
+import { storeName } from '../store';
+
+type ConfirmModalProps = {
+  onClose: () => void;
+  onConfirm: () => void;
+  confirmText: string;
+  title: string;
+  message: string | JSX.Element;
+};
+function ConfirmModal({
+  onClose,
+  onConfirm,
+  confirmText,
+  title,
+  message,
+}: ConfirmModalProps): JSX.Element {
+  return (
+    <Modal title={title} onRequestClose={onClose}>
+      <p>{message}</p>
+      <Button variant="primary" onClick={onConfirm}>
+        {confirmText}
+      </Button>
+      <Button variant="tertiary" onClick={onClose}>
+        {__('Cancel', 'mailpoet')}
+      </Button>
+    </Modal>
+  );
+}
```

Comment:
In some places, we're using `import { __experimentalConfirmDialog as ConfirmDialog } from '@wordpress/components';`, but I guess no big deal to keep it as it is now.

Reply:
Update: I think this was done later.